ruby/example_code/s3/s3.rb

```diff
@@ -0,0 +1,74 @@
+# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I added waiters for create/delete bucket ops and committed the new file to master.